external/llvm-project/mlir/lib/Dialect/Tosa/Transforms/TosaPartition.cpp

```diff
@@ -115,9 +116,7 @@ bool isElementwiseOp(Operation *op) {
   // clang-format on
 }
 
-bool isFuseableOp(Operation *op) {
-  return isElementwiseOp(op) || isa<tosa::TransposeOp, tosa::ReshapeOp>(op);
-}
+bool isFuseableOp(Operation *op) { return isElementwiseOp(op); }
```

Comment:
Could just replace isFuseableOp with isElementwiseOp.

Reply:
I like the naming for now.